examples/io/elekta_average.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
I think I will be able to dig up a simple multimodal raw .fif relatively quickly. I just need to acquire the permission to use it.